ui/src/flux/constants/functions.ts

```diff
@@ -205,12 +431,34 @@ export const FUNCTIONS: FluxToolbarFunction[] = [
         type: 'String',
       },
     ],
+    package: '',
     desc: 'Computes the covariance between two columns.',
     example:
       'covariance(columns: ["column_x", "column_y"], pearsonr: false, valueDst: "_value")',
     category: 'Aggregates',
     link:
-      'https://docs.influxdata.com/flux/latest/functions/transformations/aggregates/covariance',
+      'https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/covariance/',
+  },
+  {
+    name: 'csv.from',
+    args: [
+      {
+        name: 'file',
```

Comment:
I assume `csv.from(file: ...)` will work with v1, but I'm not 100% sure. `csv.from(csv: ...)` will work.